src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -1587,9 +1587,17 @@ static int client_handle_forcerenew(sd_dhcp_client *client, DHCPMessage *force,
         if (r != DHCP_FORCERENEW)
                 return -ENOMSG;
 
+#if 0
         log_dhcp_client(client, "FORCERENEW");
 
         return 0;
+#else
+        /* Ignore FORCERENEW requests until we implement RFC3118 (Authentication for DHCP Messages)
```

Comment:
Maybe add a `FIXME`? Are there issues open to implement these RFCs, to track this?